File: documentation/user/en/solve/examples/render-category-menu/mega-menu.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				require(
					hierarchyOfReference(
						"categories",
						fromRoot(
							"megaMenu",
							entityFetch(
								attributeContent("code")
							),
							stopAt(
								level(2)
							)
						)
					),
					page(1, 0)
				)
			)
		);
	}
);

File: documentation/user/en/solve/examples/render-category-menu/mega-menu.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	Require(
        		HierarchyOfReference(
        			"categories",
        			RemoveEmpty,
        			FromRoot(
        				"megaMenu",
        				EntityFetch(
        					AttributeContent("code")
        				),
        				StopAt(
        					Level(2)
        				)
        			)
        		),
        		Page(1, 0)
        	)
        )
	)
);